### DISCUSSION TITLE
Feature Request: Additional movement (left/right) when looking back.

### DISCUSSION THREAD

**Discussion by djesse** [opentrack/opentrack#1855] [Ideas]
Hi,

I remember a very nice feature in FaceTrackNoIR that helped alot when flying old WW2 planes in DCS. When looking back (90-180°) you could configure an automatic transitional movement of your body to left or right that made it a lot easier to check whats right behind your plane. When you only turn your head, you see the  back of your seat. By adding the automatic left / right transition, the head moved into the direction of the turn and came closer to the cockpit window, which allowed for a better view. I hope, this explanation makes sence.

I'd love to see such a feature here in OpenTrack as well.

**Comment by LaysClassicO** [comment]
You can do this if you use the input of the x output as yaw.
